lib/sidekiq/testing.rb

```diff
@@ -143,28 +202,36 @@ module Worker
     #
     module ClassMethods
 
+      # Queue for this worker
+      def queue
+        self.sidekiq_options["queue"]
```

Comment:
I think we're safe b/c of the underlying hash: https://github.com/mperham/sidekiq/blob/queue-testing-api/lib/sidekiq/worker.rb#L81